qiskit/aqua/circuits/fourier_transform_circuits.py

```diff
@@ -86,6 +91,18 @@ def construct_circuit(
             FourierTransformCircuits._do_swaps(circuit, qubits)
 
         qubit_range = reversed(range(len(qubits))) if inverse else range(len(qubits))
+
+        if precedes_measurement and not do_swaps and not inverse and approximation_degree=0:
```

Comment:
@gillenhaalb did you mean `==` rather than `=`?

Reply:
Whoops, yes absolutely, thanks. Sorry it's so bare bones at the moment too--I'm currently working on getting the swaps, inverse, and approximate circuits figured out, but unfortunately the advocate applications are due before I have time to finish it all. Not my proudest moment, but had to push this limited version to get the "contribution" requirement in the meantime.